drivers/pinmux/stm32/pinmux_stm32.c

```diff
@@ -151,8 +151,8 @@ int stm32_dt_pinctrl_configure(const struct soc_gpio_pinctrl *pinctrl,
 #ifdef CONFIG_PM_DEVICE_IDLE
 		/* FIXME: Use of device_pm clock mgmt function */
 		/* breaks correct gpio driver behavior */
```

Comment:
`FIXME` comments likely need to go away now

Reply:
yep, just changed to you be able to check / confirm